kahuna/public/js/image/stream.js

```diff
@@ -0,0 +1,48 @@
+import Rx from 'rx';
+
+function imageStream(images_) {
+
+    const images = new Set();
+    const images$ = new Rx.BehaviorSubject([]);
+
+    if (images_) {
+        images_.forEach(image => images.add(image));
```

Comment:
It could but I didn't want to have a
```JavaScript
if (images_) {
   const images = new Set(images_)
} else {
  const images = new Set();
}
```

Seems to defeat the point of `const`.